extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -551,7 +552,7 @@ public String getFormatString()
     objectMapper.registerModules(sqlModule.getJacksonModules());
     objectMapper.registerModules(BuiltInTypesModule.getJacksonModulesList());
 
-    doReturn(mock(Request.class)).when(brokerClient).makeRequest(any(), anyString());
+    doReturn(mock(Request.class)).when(brokerClient).submitSqlQuery(any(ClientSqlQuery.class));
```

Comment:
```suggestion
```

Reply:
Usages of `brokerClient` can be completely removed from here. Tests are failing because of this.